src/src/com/microsoft/aad/adal/AuthenticationSettings.java

```diff
@@ -253,4 +255,31 @@ public void setReadTimeOut(int timeOutMillis) {
 
         this.mReadTimeOut = timeOutMillis;
     }
+
+    /**
+     * Method to enable/disable WebView hardware acceleration used in
+     * {@link AuthenticationActivity} and {@link AuthenticationDialog}. 
+     * By default hardware acceleration is enable in WebView.
+     *  
+     * @param enable
+     *             if true, WebView would be hardwareAccelerated else it 
+     *             would be disable.
+     *             
+     * @see #isWebViewHardwareAccelerated()
+     */
+    public void setWebViewHardwareAcceleration(boolean enable) {
```

Comment:
Can we have the the name as setEnableHardwareAcceleration(boolean enabled)/getEnableHardwareAcceleration?